pandas/tests/arrays/string_/__init__.py

```diff
@@ -0,0 +1 @@
+arrow_string_storage = ("pyarrow",)
```

Comment:
Could this be put in a conftest.py instead? IMO variables are kinda hard to discover here

Reply:
Do you want to use as a fixture or as a variable from conftest? Not sure how I feel about using this in every test function signature

Reply:
Was thinking as a fixture. Is it needed in all these string tests currently?

Reply:
in a bunch of them, inheritance makes it a bit ugly though